extensions/ql-vscode/test/vscode-tests/no-workspace/data/remote-queries/queries/MRVA Integration test 1-6sBi6oaky_fxqXW2NA4bx/11111111/results.sarif

```diff
@@ -64,7 +64,7 @@
stom label for this query, interpolate and use that.\n   * Otherwise, use the name of the query.\n   *\n   * @returns the name of the query, unless there is a custom label for this query.\n   */\n  getShortLabel(): string {\n    return this.initialInfo.userSpecifiedLabel\n      ? this.interpolate(this.initialInfo.userSpecifiedLabel)\n      : this.getQueryName();\n  }\n\n  /**\n   * The query's file name, unless it is a quick eval.\n   * Queries run through quick evaluation are not usually the entire query file.\n   * Label them differently and include the line numbers.\n   */\n  getQueryFileName() {\n    if (this.initialInfo.quickEvalPosition) {\n      const { line, endLine, fileName } = this.initialInfo.quickEvalPosition;\n      const lineInfo = line === endLine ? `${line}` : `${line}-${endLine}`;\n      return `${path.basename(fileName)}:${lineInfo}`;\n    }\n    return path.basename(this.initialInfo.queryPath);\n  }\n\n  /**\n   * Three cases:\n   *\n   * - If this is a completed query, use the query name from the query metadata.\n   * - If this is a quick eval, return the query name with a prefix\n   * - Otherwise, return the query file name.\n   */\n  getQueryName() {\n    if (this.initialInfo.quickEvalPosition) {\n      return 'Quick evaluation of ' + this.getQueryFileName();\n    } else if (this.completedQuery?.query.metadata?.name) {\n      return this.completedQuery?.query.metadata?.name;\n    } else {\n      return this.getQueryFileName();\n    }\n  }\n\n  get completed(): boolean {\n    return !!this.completedQuery;\n  }\n\n  completeThisQuery(info: QueryWithResults) {\n    this.completedQuery = new CompletedQueryInfo(info);\n\n    // dispose of the cancellation token source and also ensure the source is not serialized as JSON\n    this.cancellationSource?.dispose();\n    delete this.cancellationSource;\n  }\n\n  /**\n   * If there is a failure reason, then this query has failed.\n   * If there is no completed query, then this query is still running.\n   * If there is a completed query, then check if didRunSuccessfully.\n   * If true, then this query has completed successfully, otherwise it has failed.\n   */\n  get status(): QueryStatus {\n    if (this.failureReason) {\n      return QueryStatus.Failed;\n    } else if (!this.completedQuery) {\n      return QueryStatus.InProgress;\n    } else if (this.completedQuery.didRunSuccessfully) {\n      return QueryStatus.Completed;\n    } else {\n      return QueryStatus.Failed;\n    }\n  }\n\n  /**\n   * The `config` property must not be serialized since it contains a listerner\n   * for global configuration changes. Instead, It should be set when the query\n   * is deserialized.\n   *\n   * @param config the global query history config object\n   */\n  setConfig(config: QueryHistoryConfig) {\n    // avoid serializing config property\n    Object.defineProperty(this, 'config', {\n      enumerable: false,\n      writable: false,\n      configurable: true,\n      value: config\n    });\n  }\n}\n"
```

Comment:
I don't think we need to change the names inside of these SARIF files. These SARIF files are just used for tests and it doesn't matter whether the contents matches with the actual source code.

I think it'd be better to not make changes to these files, unless absolutely necessary.